### ISSUE TITLE
No Hover Data Returned For OpenStudio Tools bar Icons

### ISSUE THREAD

**Issue by eomere** [NREL/openstudio-sketchup-plugin#4]
Hovering over icons on tool bars typically reveals the corresponding names of each icon. This is not the case for the OpenStudio Tools bar icons. This issue is only present within the Mac Version of the Plugin.

**Comment by jmarrec** [comment]
So you're not seeing this (Win 10, OS 2.9.1) on Mac?

![image](https://user-images.githubusercontent.com/5479063/77428492-71554a80-6dd8-11ea-832a-216a843ce335.png)

**Comment by eomere** [comment]
Yes, I am not seeing that on the Mac for SketchUp Pro 2019's OpenStudio App 1.0 Plugin.

**Comment by jmarrec** [comment]
@tijcolem Is Sketchup 2019 the officially supported Sketchup version for 3.x?
